resources/views/components/country-select.blade.php

```diff
@@ -1,9 +1,16 @@
 <graphql query="{ countries { two_letter_abbreviation full_name_locale } }" cache="countries">
     <div v-if="data" slot-scope="{ data }">
-        <x-rapidez::select {{ $attributes }}>
-            <option v-for="country in data.countries" :value="country.two_letter_abbreviation.toUpperCase()">
-                @{{ country.full_name_locale }}
-            </option>
-        </x-rapidez::select>
+        <x-rapidez::input-field.select {{ $attributes }}>
+            <x-slot:input>
+                <option
+                    v-for="country in data.countries.toSorted((a, b) =>
+                        (a.full_name_locale ?? '').localeCompare(b.full_name_locale ?? '')
+                    )"
```

Comment:
`toSorted` is still a very new function and, although it has universal support now, will cause the site to break on slightly outdated browser versions that are still relevant.